lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -243,10 +245,14 @@ sub engine_workit {
                 return {error => "Failed to send $rsync_request_description"} unless $rsync_request->enqueue;
                 log_info("Enqueued $rsync_request_description");
 
-                sleep 5 and update_setup_status until $rsync_request->processed;
+                # FIXME: pass client or job object for updating setup status
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit or ticket

Reply:
There is no disabled code. However, the 'FIXME' is just a leftover because I'm actually passing the `$job` object now. I'll remove it.